rskj-core/src/main/java/co/rsk/db/MutableTrieImpl.java

```diff
@@ -86,6 +86,15 @@ public Uint24 getValueLength(byte[] key) {
         return atrie.getValueLength();
     }
 
+    @Override
+    public Keccak256 getValueHash(byte[] key) {
+        Trie atrie = trie.find(key);
+        if (atrie == null) {
+            return Keccak256.ZERO_HASH;
```

Comment:
I think a reference here to the EIP stating that missing keys should hash to _zero_ will be greatly appreciated

Reply:
Actually RSKIP140 or [EIP](https://eips.ethereum.org/EIPS/eip-1052) specifies that if an account does not exist, zero is returned. At MutableTrieImpl level ZERO_HASH is a way to difference between Account/Contract and Non-existing account.

Reply:
BTW, i wrote this comment: _From RSKIP140 empty-account specification._